lesson_9/src/appliances/KitchenAppls.java

```diff
@@ -0,0 +1,15 @@
+package appliances;
+
+public abstract class KitchenAppls extends Appliances {
+    private int power;
+
+    public abstract void switchOn();
```

Comment:
you should have duplicate abstract methods in extended class.
switchOn in appliances will be enough